atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/config/WorkflowExecutionConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.config;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+@JsonDeserialize(as = ImmutableWorkflowExecutionConfiguration.class)
+@JsonSerialize(as = ImmutableWorkflowExecutionConfiguration.class)
+@Value.Immutable
+public interface WorkflowExecutionConfiguration {
+    RunMode runMode();
+
+    enum RunMode {
+        ONE,
```

Comment:
Is what we want for better Antithesis fuzzing that this is only one thing, but random? Or is it better if we publish separate packages for each workflow that we want to test and subdivide testing hours across them accordingly?

Reply:
That's a fair question. The thinking behind this decision on my end was to have their fuzzer tune the number of hours spent on each workflow in an attempt to find as many new edges as they can (as effectively this introduces a decision point very early in the run on which workflow to run), as compared to us choosing some static distribution (so I think the version as written is good). Unless you think we might want to split by packages?